Documentation/sourcebuild-user-onboarding-guide/installation.md

```diff
@@ -0,0 +1,27 @@
+## Installation on Linux
```

Comment:
This seems like it would be dupliacting some of https://github.com/dotnet/source-build/blob/main/Documentation/packaging-installation.md. Should we delete that other copy or somehow merge these?

Reply:
That's a good point. I will check with the team and delete one of the copy.